test/Prototypes/BigInt.swift

```diff
@@ -1871,10 +1871,24 @@ BigIntBitTests.test("words") {
   expectEqualSequence([UInt.max], (-1 as BigIntBit).words)
 }
 
-runAllTests()
-
 BigIntTests.test("isMultiple") {
   // Test that these do not crash.
   expectTrue((0 as _BigInt<UInt>).isMultiple(of: 0))
   expectFalse((1 as _BigInt<UInt>).isMultiple(of: 0))
 }
+
+BigIntTests.test("isPower") {
+  expectTrue((0 as _BigInt<UInt>).isPower(of: 0))
+  expectTrue((1 as _BigInt<UInt>).isPower(of: 0))
+  expectTrue((1 as _BigInt<UInt>).isPower(of: 1))
+  expectTrue((1 as _BigInt<UInt>).isPower(of: -1))
+  expectTrue((-1 as _BigInt<UInt>).isPower(of: -1))
+  expectFalse((-1 as _BigInt<UInt>).isPower(of: 1))
+  expectTrue((4 as _BigInt<UInt>).isPower(of: 2))
+  expectTrue((-8 as _BigInt<UInt>).isPower(of: -2))
+  expectTrue((16 as _BigInt<UInt>).isPower(of: -2))
+  expectFalse((32 as _BigInt<UInt>).isPower(of: -2))
+  expectTrue((100 as _BigInt<UInt>).isPower(of: 10))
+}
```

Comment:
The `BigInt` prototype has been `XFAIL`ed recently. Should this test be removed?

Reply:
How about we keep this test here, in case `BigInt` would be un`XFAIL`ed some time in future? Anyway, I don't mind removing it.

Reply:
It's not for me to decide, but I'd guess that's OK.

I was going to suggest testing more multi-word values, but DoubleWidth is still available for that purpose.

Reply:
Well, I reverted changes in BigInt, since DoubleWidth, as you mentioned, serves multi-words purpose already. We'd better not touch `XFAIL`'d tests unless necessary.